assets/airports/cyhz.json

```diff
@@ -0,0 +1,656 @@
+        "IGVOP":    ["N45d03m51.00", "W063d38m64.00"],
+        "JIGGS":    ["N47d25m22.00", "W059d48m10.00"],
+        "LEROS":    ["N45d01m54.00", "W063d22m13.00"],
+        "LOGPO":    ["N44d58m59.00", "W063d33m68.00"],
+        "MIXIV":    ["N45d02m10.00", "W063d35m80.00"],
+        "MODIR":    ["N44d55m32.00", "W063d46m73.00"],
+        "OBNOM":    ["N44d41m02.00", "W063d33m82.00"],
+        "ODKAS":    ["N44d43m89.00", "W063d39m55.00"],
+        "PUTUX":    ["N44d59m62.00", "W063d55m43.00"],
+        "RONPO":    ["N44d40m34.00", "W063d51m58.00"],
+        "TETAR":    ["N44d59m42.00", "W063d42m69.00"],
+        "TIDON":    ["N44d59m78.00", "W063d31m13.00"],
+        "UDNIM":    ["N44d47m75.00", "W063d07m17.00"],
+        "VOKIL":    ["N44d47m15.00", "W063d18m13.00"],
+        "YHZ":      ["N44d55m40.00", "W063d24m10.00"],
+        "XOVMU":    ["N44d55m97.00", "W063d35m76.00"],
+        "RW14":     ["N44d53m37.02", "W063d31m04.81"],
+        "TAKIM":    ["N44d52m47.00", "W063d28m74.00"],
+        "DUXIK":    ["N44d49m02.00", "W063d21m85.00"],
+        "PENLU":    ["N44d50m61.00", "W063d25m02.00"],
+        "RW32":     ["N44d52m53.97", "W063d29m38.67"],
+        "XUGID":    ["N44d54m04.00", "W063d31m89.00"],
+        "VITAV":    ["N44d57m49.00", "W063d38m81.00"],
+        "XOMRO":    ["N44d42m93.00", "W063d40m38.00"],
+        "RW05":     ["N44d52m08.44", "W063d31m28.58"],
+        "URMIN":    ["N45d00m13.00", "W063d47m11.00"],
+        "XUBKO":    ["N45d03m84.00", "W063d19m77.00"],
+        "PUTUD":    ["N44d59m74.00", "W063d13m02.00"],
+        "NUBUK":    ["N44d57m85.00", "W063d36m30.00"],
+        "RW23":     ["N44d53m18.04", "W063d30m19.91"],
+        "PAULO":    ["N45d15m20.00", "W063d20m10.00"],
+        "NOTOP":    ["N45d27m16.00", "W062d00m72.00"],
+        "NUTBY":    ["N45d41m25.00", "W063d14m80.00"],
+        "CAAPE":    ["N45d18m00.00", "W065d17m77.00"],
+        "YZX":      ["N44d55m63.00", "W065d05m12.00"],
+        "YQI":      ["N43d49m51.00", "W066d04m95.00"],
+        "YQM":      ["N46d11m34.00", "W064d34m25.00"],
+
+        "BOSUB":    ["N45d11m83.00", "W062d55m52.00"],
+        "CAYLY":    ["N45d37m27.00", "W062d14m50.00"],
+        "KEBOT":    ["N44d59m18.00", "W063d22m63.00"],
+        "VIMLA":    ["N44d58m76.00", "W063d29m09.00"],
+        "MENVO":    ["N44d53m86.00", "W063d11m68.00"],
+        "KIXIT":    ["N44d48m33.00", "W063d08m31.00"],
+        "MUVOX":    ["N44d51m85.00", "W064d12m20.00"],
+        "EBONY":    ["N44d54m14.00", "W067d09m39.00"],
+        "EBKID":    ["N44d53m42.00", "W065d30m08.00"],
+        "ALLEX":    ["N44d25m00.00", "W067d00m00.00"],
+        "LIRLA":    ["N45d01m25.00", "W062d50m13.00"],
+        "BAFDY":    ["N44d46m66.00", "W064d46m95.00"],
+        "AGMIR":    ["N45d13m70.00", "W061d57m33.00"],
+        "ETGAR":    ["N44d25m16.00", "W064d09m78.00"],
+        "ILOTA":    ["N43d33m90.00", "W067d00m00.00"],
+        "TUSKY":    ["N44d52m52.00", "W063d48m90.00"],
```

Comment:
I show TUSKY as actually being located here:
`"TUSKY":    ["N43d33m52.20", "W067d00m01.80"],`

This location makes the procedure make sense again.